gflownet/envs/crystals.py

```diff
@@ -229,7 +229,7 @@ def reset(self, env_id=None):
         self.id = env_id
         return self
 
-    def get_parents(self, state=None, done=None, actions=None):
+    def get_parents(self, state=None, done=None, action=None):
```

Comment:
This is super minor, but it would be good to update the name in docstring as well (actions -> action).

Reply:
absolutely!